### PR TITLE
Cockpit 266

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-266.html

### CODE REVIEW

**Comment by garrett** [comment]
The only PR we're mentioning this week:
- https://github.com/cockpit-project/cockpit/pull/17163

File: _posts/2022-03-30-cockpit-266.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 266:
+
+
+## Standardize on SI-based units for data sizes and rates
```
Comment (allisonkarlitskaya) [inline _posts/2022-03-30-cockpit-266.md:17]: And when exactly did we standardise on american spellings? :)

**Review by allisonkarlitskaya — APPROVED** [review]

**Comment by allisonkarlitskaya** [comment]
There's only one note and I already reviewed it when it was part of the PR.  We need a flathub release.  I'll work on that today.

File: _posts/2022-03-30-cockpit-266.md (hunk 1 above)
Comment (garrett) [reply _posts/2022-03-30-cockpit-266.md:17]: Since the beginning, as we work for an American company with a big bulk of its customers in the US?

Comment (garrett) [reply _posts/2022-03-30-cockpit-266.md:17]:
```suggestion
## Consistently use SI-based units for data sizes and rates
```

Non-regional spelling suggestion above.

Or even simpler:

```suggestion
## Use SI-based units for data sizes and rates
```

Comment (allisonkarlitskaya) [reply _posts/2022-03-30-cockpit-266.md:17]: To be clear: this comment was a joke.

**Pull request merged**